Week2/homework/part2/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const program = require('commander');
+const queries = require('./queries');
+
+const help = queries.help;
+const addTrigger = queries.addTrigger;
+const insertLanguage = queries.insertLanguage;
+
+if (!process.argv[2]) {
+  help();
+} else {
+  program
```

Comment:
The commands trigger or insert never get executed. Didn't you forget something? :) 
Tip: you need to parse the arguments...